templates/taxi/index.html

```diff
@@ -10,4 +10,5 @@ <h2>Dynamic content</h2>
     <li><strong>Drivers:</strong> {{ num_drivers }}</li>
     <li><strong>Manufacturers:</strong> {{ num_manufacturers }}</li>
   </ul>
+  <p>Times you have visited this page: {{ num_visits }}!</p>
```

Comment:
Please, use `pluralize` here as mentioned in the checklist. Update the corresponding screenshot

Reply:
In the checklist is mentioned to use the pluralize, but I changed the phrase so that using makes no sense